### PR TITLE
Finished Building portfolio mobile.

### PR DESCRIPTION
**Pull request by Caren-Koroeny**
- Worked on about me `section`.
- Created a 'form'  for contact section.
- Used `grid` in main section.
-  Use `Flexbox` to lay out an individual card.
- Used  `ul` and `li`  for categories in `Work` section, for `Html` , `Css` , and `Javascript` lists.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,88 +1,320 @@
...
+        </p>
+        <br>
+        <ul class="language-categories">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="submit-btn" type="submit">See Project</button>
+      </div>
+
```
Comment (zieeco) [inline index.html]: - Hey great job with the design, however, your images are not showing. let's correct the file path as that's why there are not showing up. Update this file path `` to this ` ` and it should fix them. 


:point_right:  *This is for all your images*

---

File: index.html [hunk 2]
```diff
@@ -1,88 +1,320 @@
...
+          <li class="text-2">
+            <p>Back&nbspEnd&nbspDev</p>
+          </li>
+          <img src="/images/counter-images.png" alt="" class="counter-image">
+          <li class="text-3">2015</li>
+        </ul>
+
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
```
Comment (zieeco) [inline index.html:105]: - Let's align the paragraph correctly by adding the text-align: left to improve our paragraph on all the cards so as to match the design as much as you can.

File: index.html [hunk 3]
```diff
@@ -1,88 +1,320 @@
...
+        <br>
+        <ul class="language-categories">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="submit-btn" type="submit">See Project</button>
```
Comment (zieeco) [inline index.html:112]: - Let's also add the missing interaction between buttons by indicating when it's active, hovered, pressed, etc. kindly check the Figma design to see the correct color used to improve our design.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Caren-Koroeny ,


Great job so far! 👏 
You've done it well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -1,88 +1,320 @@
...
+        
+        <ul class="project-badges">
+          <li class="text-1">Canopy</li>
+          <img src="/images/counter-images.png" alt="" class="counter-image">
+          <li class="text-2">
+            <p>Back&nbspEnd&nbspDev</p>
+          </li>
+          <img src="/images/counter-images.png" alt="" class="counter-image">
+          <li class="text-3">2015</li>
+        </ul>
```
Comment (binhussen) [inline index.html]: - kindly, make sure your image URL is correct as your previous code reviewer suggested, if you are accessing the image from the same folder use `./` not `/` 👍 👍 and use all of your images.😍

File: index.html [hunk 5]
```diff
@@ -1,88 +1,320 @@
...
+      <div class="left-card1-block">
+        <h1 class="project-title">Multi-Post <br> Stories</h1>
+        <!-- Flex -->
+        <ul class="project-badges">
+          <li class="text-1">Canopy</li>
+          <img src="/images/counter-images.png" alt="" class="counter-image">
+          <li class="text-2">
+            <p>Back&nbspEnd&nbspDev</p>
+          </li>
+          <img src="/images/counter-images.png" alt="" class="counter-image">
```
Comment (binhussen) [inline index.html]: - kindly, make sure your image URL is correct as your previous code reviewer suggested, if you are accessing the image from the same folder use `./` not `/` 👍 👍 and use all of your images.😍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @Caren-Koroeny , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
- Please, check all images are visible and cross chek the URL as your previous reviewer suggested you.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Caren-Koroeny 
Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- design ok
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**